lametro/views.py

```diff
@@ -282,7 +282,10 @@ def get_context_data(self, **kwargs):
         end_date_str   = self.request.GET.get('to')
         day_grouper    = lambda x: (x.local_start_time.year, x.local_start_time.month, x.local_start_time.day)
 
-        minutes_queryset = EventDocument.objects.filter(note__icontains='minutes')
+        # We only want to display approved minutes
+        all_event_docs = EventDocument.objects.select_related('event')
+        approved_minutes = [doc.id for doc in all_event_docs if doc.event.extras.get('approved_minutes')]
+        minutes_queryset = all_event_docs.filter(id__in=approved_minutes)
```

Comment:
This will give you all documents related to events that have approved minutes. It's also possible to do all of this in a single ORM statement. Can you add back the original note condition, and refactor this into one query? LMK if you need any hints.

Reply:
Great points, thanks!